WARNING:root:You are using solc version 0.4.26. The supported version is 0.4.21.
INFO:root:Contract /sb/0x871a71438029549a836c042d3bd0e88ca6c88388.sol:Matrix:
INFO:symExec:Running, please wait...
Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 315, in 'calling callback function'
  File "/root/osiris/z3/z3.py", line 137, in z3_error_handler
    def z3_error_handler(c, e):
  File "/root/osiris/symExec.py", line 2868, in handler

  .oooooo.             o8o            o8o          
 d8P'  `Y8b            `"'            `"'          
888      888  .oooo.o oooo  oooo d8b oooo   .oooo.o
888      888 d88(  "8 `888  `888""8P `888  d88(  "8
888      888 `"Y88b.   888   888      888  `"Y88b. 
`88b    d88' o.  )88b  888   888      888  o.  )88b
 `Y8bood8P'  8""888P' o888o d888b    o888o 8""888P'


!!! SYMBOLIC EXECUTION TIMEOUT !!!
    raise Exception("timeout")
Exception: timeout
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 53.1%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 16.0240440369 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract /sb/0x871a71438029549a836c042d3bd0e88ca6c88388.sol:Ownable:
INFO:symExec:Running, please wait...
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 99.4%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 0.175832986832 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract /sb/0x871a71438029549a836c042d3bd0e88ca6c88388.sol:SafeMath:
INFO:symExec:Running, please wait...
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 100.0%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 0.00578308105469 seconds ---
INFO:symExec:	====== Analysis Completed ======
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored
Exception z3.z3types.Z3Exception: Z3Exception() in <object repr() failed> ignored


